assets/src/activation-notice/app/components/step1.js

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import { useCallback } from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { __, _x } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { trackClick } from '../../../tracking';
+import { useConfig } from '../config';
+
+import Paragraph from './paragraph';
+import Link from './link';
+import Number from './number';
+
+const Wrapper = styled.div`
+  display: none;
+  min-width: 300px;
+
+  @media ${({ theme }) => theme.breakpoint.tabletSmall} {
+    display: flex;
+  }
+`;
+
+const Image = styled.img`
+  transform: rotate(10.57deg);
+  margin-top: -40px;
+`;
+
+const ParagraphWrapper = styled.div`
+  align-self: flex-end;
+  margin-bottom: 15px;
+`;
+
+// @todo Support markup in translated strings - https://github.com/google/web-stories-wp/issues/1578
+function Step1() {
+  const { assetsURL, demoStoryURL } = useConfig();
+
+  const onClick = useCallback(
+    (evt) => {
+      trackClick(evt, 'activation-notice', demoStoryURL);
+    },
+    [demoStoryURL]
+  );
+
+  return (
+    <Wrapper>
+      <Link href={demoStoryURL} onClick={onClick}>
+        <Image
+          src={`${assetsURL}images/plugin-activation/tips.png`}
+          alt={__('Screenshot of the Get Started Story', 'web-stories')}
+          width={170}
+          height={300}
+        />
+      </Link>
+      <ParagraphWrapper>
+        <Number>{1}</Number>
```

Comment:
Should these numbers be translatable as well? There are langauges that use different characters for number rights?

Reply:
Hmm yeah makes sense. Will add.